app/views/brexit_checker/results.html.erb

```diff
@@ -84,7 +84,7 @@
       } %>
     <% end %>
 
-    <%= render 'results_business_actions', business_results: @presenter.business_results if @presenter.business_results.any? %>
+    <%= render 'results_business_actions' if @presenter.business_results.any? %>
```

Comment:
What was wrong with this?

Reply:
Nothing. But I want to be able to show either ungrouped(`@presenter.business_results`) or grouped (`@presenter.business_results_groups`) depending on the presence of the flag. I thought it was easier to put the if/else logic in the results_business_actions partial rather than here.